algoliasearch-apache-uber/pom.xml

```diff
@@ -44,8 +44,13 @@
                     <instructions>
                         <Bundle-SymbolicName>com.algolia.algoliasearch-apache-uber</Bundle-SymbolicName>
                         <Bundle-Name>algoliasearch-apache-uber</Bundle-Name>
-                        <Export-Package>com.algolia.search</Export-Package>
-                        <Import-Package>javax.net.ssl,javax.naming,*</Import-Package>
+                        <Bundle-Version>${project.version}</Bundle-Version>
+                        <Export-Package>com.algolia.search.*,com.algolia.search.org.apache.http.impl.nio.client</Export-Package>
+                        <Import-Package>
+                            javax.net.ssl,javax.naming,!org.apache.avalon.framework.logger,!org.apache.log,*
```

Comment:
How did we find those import package?

Reply:
By generating the uber JAR with `mvn package` and installing it to the OSGi/AEM instance and see which imports were missing there, one by one.

Reply:
Huge work! Thanks.